README.md

```diff
@@ -88,6 +88,5 @@ Tomatoes is released under the [MIT license](https://raw.github.com/tomatoes-app
 
 [![Build Status](https://travis-ci.org/tomatoes-app/tomatoes.svg?branch=master)](https://travis-ci.org/tomatoes-app/tomatoes)
 [![Code Climate](https://codeclimate.com/github/tomatoes-app/tomatoes/badges/gpa.svg)](https://codeclimate.com/github/tomatoes-app/tomatoes)
-[![Test
-Coverage](https://codeclimate.com/github/tomatoes-app/tomatoes/badges/coverage.svg)](https://codeclimate.com/github/tomatoes-app/tomatoes/coverage)
+[![Coverage Status](https://coveralls.io/repos/github/tomatoes-app/tomatoes/badge.svg?branch=coveralls)](https://coveralls.io/github/tomatoes-app/tomatoes?branch=coveralls)
```

Comment:
I think so, I copied it from the coveralls documentation and without it doesn't seem to work. :thinking: